scripts/redis/scan_redis.py

```diff
@@ -60,10 +72,27 @@ def run():
         for key in data:
             key_type = client.type(key)
             size = get_size(client, key, key_type)
+
+            # ttl() returns None in redis 2.x and -1 in redis 3.x for
+            # keys that don't have an expiration.
```

Comment:
That's a weird interface. I'd just fix it right here with `if ttl == -1: ttl = None` instead of wondering how it impacts the code down the line.